cmd/necoperf-daemon/cmd/daemon.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"log/slog"
+	"os"
+
+	"github.com/cybozu-go/necoperf/pkg/daemon"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(daemonCmd)
+	flags := daemonCmd.Flags()
+	flags.IntP("port", "p", 6543, "Set server port number")
+	flags.StringP("runtime-endpoint", "r", "unix:///run/containerd/containerd.sock", "Set container runtime endpoint")
```

Comment:
I prefer `IntVar`, `StringVar` to `IntP`, `StringP`.
It is safer to refer by a variable than by a string.